models/contacts.js

```diff
@@ -1,19 +1,96 @@
-// const fs = require('fs/promises')
+const fs = require("fs/promises");
+const crypto = require("crypto");
+const path = require("path");
 
-const listContacts = async () => {}
+const contactsPath = path.join(__dirname, "contacts.json");
 
-const getContactById = async (contactId) => {}
+/**
+ * The function `listContacts` reads a file containing contacts and returns them as a parsed JSON
+ * object.
+ * @returns The function `listContacts` returns a promise that resolves to an array of contacts.
+ */
 
-const removeContact = async (contactId) => {}
+const listContacts = async () => {
+  const contacts = await fs.readFile(contactsPath);
+  return JSON.parse(contacts);
+};
 
-const addContact = async (body) => {}
+/**
+ * The function `getContactById` retrieves a contact object from a list of contacts based on the
+ * provided contactId.
+ * @param contactId - The `contactId` parameter is the unique identifier of the contact that we want to
```

Comment:
```suggestion
 * @param contactId - The `contactId` parameter is the unique identifier of the contact that we want to
``` переглянте доки jsdoc там також можна вказувати тип даниx e.g. @param {string} і тп.